cms/djangoapps/contentstore/tests/test_course_settings.py

```diff
@@ -942,6 +943,16 @@ def test_allow_unsupported_xblocks(self):
         XBlockStudioConfigurationFlag(enabled=True).save()
         self.assertIn('allow_unsupported_xblocks', CourseMetadata.fetch(self.fullcourse))
 
+    def test_allow_adaptive_learning_configuration(self):
+        """
+        Tests to make sure that depending on how the flag is set, whether or not
+        the the adaptive_learning_configuration is located in the filtered list.
```

Comment:
@macornwell This sentence is a little convoluted. How about we (roughly) follow the pattern of the [previous docstring](https://github.com/open-craft/edx-platform/blob/2f27e2cfb7ce2a55ff02c2cfde032a8e2575d916/cms/djangoapps/contentstore/tests/test_course_settings.py#L939-L940) instead:

```
Test that `adaptive_learning_configuration` is only shown in Advanced Settings
if AdaptiveLearningEnabledFlag is enabled.
```

Reply:
You're clearly a better writer than me. Sounds good!